CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.4.0 (In Progress)
+===================
+
+New Features
+------------
+
+- In-Band-Turn - Previously twilio-video used to send an http request to Endpoint Configuration Service (ECS) to get RTCIceServers before connecting to a Room. It will now get the RTCIceServers directly from the Room Service. This simplifies the connection flow and does not need these [additional connectivity requirements](https://www.twilio.com/docs/video/ip-address-whitelisting#additional-connectivity-requirements) anymore. (JSDK-2675)
```

Comment:
I would only include public facing services mentioned in additional connectivity requirements, and omit any mention of Room Service. Drafted here:

- In-Band-Turn - Previously, twilio-video used to send an https request to ecs.us1.twilio.com to fetch RTCIceServers before connecting to a Room. Now the SDK will fetch RTCIceServers from the signaling edge (*.vss.twilio.com). This simplifies the connection flow, and allowing some operations to be performed in parallel, and removes the need for the [additional connectivity requirements](https://www.twilio.com/docs/video/ip-address-whitelisting#additional-connectivity-requirements).